src/features/click-to-play.js

```diff
@@ -606,425 +603,419 @@ const styles = {
 }
 
 /*********************************************************
- *  config (temporarily here, until localized)
+ *  config set by getConfig() at init per localization
  *********************************************************/
-const config = {
-    Facebook: {
-        informationalModal: {
-            icon: blockedFBLogo,
-            messageTitle: 'Logging in with Facebook lets them track you',
-            messageBody: "Once you're logged in, DuckDuckGo can't block Facebook content from tracking you on this site.",
-            confirmButtonText: 'Log In',
-            rejectButtonText: 'Go back'
-        },
-        elementData: {
-            'FB Like Button': {
-                selectors: [
-                    '.fb-like'
-                ],
-                replaceSettings: {
-                    type: 'blank'
-                }
-            },
-            'FB Button iFrames': {
-                selectors: [
-                    "iframe[src*='://www.facebook.com/plugins/like.php']",
-                    "iframe[src*='://www.facebook.com/v2.0/plugins/like.php']",
-                    "iframe[src*='://www.facebook.com/plugins/share_button.php']",
-                    "iframe[src*='://www.facebook.com/v2.0/plugins/share_button.php']"
-                ],
-                replaceSettings: {
-                    type: 'blank'
-                }
-            },
-            'FB Save Button': {
-                selectors: [
-                    '.fb-save'
-                ],
-                replaceSettings: {
-                    type: 'blank'
-                }
-            },
-            'FB Share Button': {
-                selectors: [
-                    '.fb-share-button'
-                ],
-                replaceSettings: {
-                    type: 'blank'
-                }
+let config = {}
+
+function getConfig (locale) {
+    const locales = JSON.parse(localesJSON)
+    const fbStrings = locales[locale]['facebook.json']
+
+    return {
+        Facebook: {
+            informationalModal: {
+                icon: blockedFBLogo,
+                messageTitle: `${fbStrings.informationalModalMessageTitle.title}`,
```

Comment:
If these are all simple strings there's no need to wrap them in `` `${}` ``, you can just have the variable.
```suggestion
                messageTitle: fbStrings.informationalModalMessageTitle.title,
```

Reply:
Ah good catch, yes templates don't really add anything here!